command/plugin_deregister.go

```diff
@@ -33,7 +34,7 @@ Usage: vault plugin deregister [options] TYPE NAME
 
   Deregister the plugin named my-custom-plugin:
 
-      $ vault plugin deregister auth my-custom-plugin
+      $ vault plugin deregister auth my-custom-plugin [version]
```

Comment:
:+1: Saw you did that in a follow-up commit. I think I was trying to avoid the `version` / `plugin_version` confusion, but it's not as relevant here as it is for mounts.